src/tile/worker.cc

```diff
@@ -0,0 +1,331 @@
+#include "tile/worker.h"
+#include "baldr/graphreader.h"
+#include "baldr/tilehierarchy.h"
+#include "meili/candidate_search.h"
+#include "midgard/constants.h"
+#include "midgard/logging.h"
+#include "valhalla/exceptions.h"
+
+#include <boost/geometry.hpp>
+#include <boost/geometry/geometries/box.hpp>
+#include <boost/geometry/geometries/linestring.hpp>
+#include <boost/geometry/geometries/multi_linestring.hpp>
+#include <boost/geometry/geometries/point_xy.hpp>
+#include <vtzero/builder.hpp>
+
+#include <algorithm>
+#include <climits>
+#include <cmath>
+#include <unordered_set>
+
+using namespace valhalla;
+using namespace valhalla::midgard;
+using namespace valhalla::baldr;
+
+namespace valhalla {
+namespace tile {
+
+tile_worker_t::tile_worker_t(const boost::property_tree::ptree& config,
```

Comment:
it is quite messy, please don't judge a lot 😄

Reply:
we can also mark it BETA for now, so no one should expect that it's really polished (at least in terms of performance or so)